### PR TITLE
Packs 3 1

### PR DESCRIPTION
**Pull request by ritawatson**
This PR 
- adds doc pages for MicroK8s, MySQL Operator, MinIO Operator, and Longhorn.
- updates versions for 
      External Secrets Operator  - 0.6.0
      Prometheus Operator - 37,2 => 40.5.0
      Nginx Ingress - 1.3 => 1.4
      Vault - 0.20.1 => 0.22.0
      EBS CSI - 1.8 =>1.12

Feel free to add any reviewers I might have missed.

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1057#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@ritawatson: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 00fafa1c301688f102f67cc141e7c9dd3ba47fcd | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1057/librarium_build/1605379282115235840) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/06-integrations/00-minio-operator.md
Comment (karl-cardenas-coding) [reply content/docs/06-integrations/00-minio-operator.md:31]: I agree, we really don't need a hyphen here @ritawatson

File: content/docs/06-integrations/00-aws-ebs.md
Comment (karl-cardenas-coding) [reply content/docs/06-integrations/00-aws-ebs.md:30]: Also, let's fix the URLs so that they are in markdown form with text visible vs raw URLs

File: content/docs/06-integrations/00-microk8s.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-microk8s.md:32]:
```suggestion
- At least 20 GB of disk space and 4 GB of memory.
```

File: content/docs/06-integrations/00-minio-operator.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-minio-operator.md:38]: @ritawatson I'll defer to you but this seems a bit different from the other new packs you created. I'm referring to the content section.

**Review by karl-cardenas-coding — APPROVED** [review]
Just some minor feedback and questions but nothing major.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1057#pullrequestreview-1226239890" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1057#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/06-integrations/00-longhorn.md
Comment (brathina-spectro) [inline content/docs/06-integrations/00-longhorn.md:25]: Min version should be 1.22 as Palette stopped supporting older versions of k8s

File: content/docs/06-integrations/00-longhorn.md
Comment (brathina-spectro) [inline content/docs/06-integrations/00-longhorn.md:19]: We should skip talking about kubectl apply or Helm chart as that is handled by the pack installation now in Palette

**Review by zulfilee — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1057#pullrequestreview-1226239890" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1057#" title="Author self-approved">ritawatson</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1057#pullrequestreview-1226465082" title="Approved">zulfilee</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-1/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/06-integrations/00-minio-operator.md
Comment (ritawatson) [reply content/docs/06-integrations/00-minio-operator.md:31]: Fixed.

File: content/docs/06-integrations/00-aws-ebs.md
Comment (ritawatson) [reply content/docs/06-integrations/00-aws-ebs.md:30]: @karl-cardenas-coding This was all original doc content - I'll make the fixes.

File: content/docs/06-integrations/00-microk8s.md
Comment (ritawatson) [reply content/docs/06-integrations/00-microk8s.md:32]: Fixed

File: content/docs/06-integrations/00-minio-operator.md
Comment (ritawatson) [reply content/docs/06-integrations/00-minio-operator.md:38]: @karl-cardenas-coding This convention was used in another existing doc and so I used it here. Renaming it to 'MinIO Operator Templates'.

File: content/docs/06-integrations/00-longhorn.md
Comment (ritawatson) [reply content/docs/06-integrations/00-longhorn.md:25]: @brathina-spectro Fixed

File: content/docs/06-integrations/00-longhorn.md
Comment (ritawatson) [reply content/docs/06-integrations/00-longhorn.md:19]: @brathina-spectro Removed mention of kubectl apply and Helm chart.

File: content/docs/06-integrations/00-external-secrets-operator.md
Comment (roshanirathi) [inline content/docs/06-integrations/00-external-secrets-operator.md:38]: The version is 0.6.0

**Review by roshanirathi — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1057#pullrequestreview-1226239890" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1057#" title="Author self-approved">ritawatson</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1057#pullrequestreview-1226641544" title="Approved">roshanirathi</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1057#pullrequestreview-1226465082" title="Approved">zulfilee</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-1/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by brathina-spectro — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1057#pullrequestreview-1226815529" title="Approved">brathina-spectro</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1057#pullrequestreview-1226239890" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1057#" title="Author self-approved">ritawatson</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1057#pullrequestreview-1226641544" title="Approved">roshanirathi</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1057#pullrequestreview-1226465082" title="Approved">zulfilee</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-1/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/06-integrations/00-external-secrets-operator.md
Comment (ritawatson) [reply content/docs/06-integrations/00-external-secrets-operator.md]: Fixed

**Pull request merged**
